connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -492,7 +492,7 @@ public boolean commitOffsets() {
             // to persistent storage
 
             // Next we need to wait for all outstanding messages to finish sending
-            log.info("{} flushing {} outstanding messages for offset commit", this, outstandingMessages.size());
+            log.debug("{} flushing {} outstanding messages for offset commit", this, outstandingMessages.size());
```

Comment:
I'm not convinced that changing this to DEBUG is worth it. The number of source records output in this log message can be instrumental in some cases.

For example, if the producer does not keep up with the source task (for whatever reason), this currently INFO-level message appears shortly before the following ERROR-level message:
```
... ERROR WorkerSourceTask{id=...} Failed to commit offsets (org.apache.kafka.connect.runtime.SourceTaskOffsetCommitter)
```
The number of outstanding source records reported by this line is an important factor in determining how to tune the producer and `offset.flush.timeout.ms` value.

Reply:
I do not feel strongly about this particular message -- it represents about 2% of the connect log volume in my deployment, so not too bad. Are you still onboard with the other log levels I've updated in the PR? In which case, I will revert this one?

Reply:
Yeah, I think the log messages other than this one are fine as DEBUG.

Reply:
Sounds good, updated